File: test/while.cpp
#include <ostream>
void nWhile();

void doWhile();

void nWhile()
{
  int limit = 15;
  int total = 0;
  int i = 1;
  while(i <= limit){
    total += i * 3;
    i += 2;
  }
}

void doWhile()
{
  int product = 1;
  int i = 1;
  int limit = 10;
  while(true){
    product *= i;
    i += 2;
    if (i > limit) {
      break;
    }
  }
}

File: test/while.py
@transpile
def nWhile():
    limit = 15
    total = 0
    i = 1
    while i <= limit:
        total += i * 3
        i += 2
@transpile
def doWhile():
    product = 1
    i = 1
    limit = 10
    while True:
        product *= i
        i += 2
        if i > limit:
            break




